myst_nb/converter.py

```diff
@@ -248,6 +258,19 @@ def _flush_markdown(start_line, token, md_metadata):
         if token.type == "fence" and token.info.startswith(code_directive):
             _flush_markdown(md_start_line, token, md_metadata)
             options, body_lines = read_fenced_cell(token, len(notebook.cells), "Code")
+            # Parse :file or file: tags and populate body with contents of file
+            if "file" in options:
+                fl = re.search(r"( .*?\.[\w:]+)", token.content).group(0).lstrip()
+                flpath = Path(context.srcdir + "/" + fl).resolve()
```

Comment:
this would fail if context was None, and anyway as above this function should not be dependent on sphinx.

Reply:
@chrisjsewell I have removed `context` passing and added `file_srcdir` to support the `file` import option.

Reply:
I need to use more than one context based variable so I have renamed the context to `file_context` to indicate it is used by the `:file:` option to `code-cell`. That will make it easier to remove if this get's implemented elsewhere.